managers/opportunity/index.js

```diff
@@ -58,6 +58,25 @@ class opportunityManager {
       throw err;
     }
   }
+
+
+  async updateOpportunity(
+    queryObject,
+    updatingprops
+  ){
+
+
+    try {
+      let updatedOpportunity = await this.opportunity.updateOne(queryObject,{
+        $set:updatingprops
+      });
+      return updatedOpportunity;
```

Comment:
will this verify the updated data before saving it to the database?

Reply:
@abdus  Currently this is not sending updated data only sending modified count, Do we need to return updated data......?

Reply:
returning the updated opportunity will be enough I think